draft-ietf-secevent-token.xml

```diff
@@ -164,27 +164,27 @@
           <t hangText="Security Event Token (SET)">
 	    <vspace/>
             A SET is a JWT <xref target="RFC7519"/> conforming to this specification
-	    that is distributed to one or more Event Recipients.
+	    that is distributed to one or more SET recipients.
           </t>
           
-          <t hangText="Event Issuer">
+          <t hangText="SET Issuer">
 	    <vspace/>
-            A service provider that creates SETs to be sent to other providers known
-            as Event Recipients. 
+            A service provider that creates SETs to be sent to other service providers known
+            as SET recipients. 
           </t>
           
-          <t hangText="Event Recipient">
+          <t hangText="SET Recipient">
 	    <vspace/>
-            An Event Recipient is an entity that receives SETs through 
-            some distribution method. An Event Recipient is the same
-            entity referred as a "recipient" or "receiver" in <xref target="RFC7519"/> and
+            A SET recipient is an entity that receives SETs through 
+            some distribution method. A SET recipient is the same
+            entity referred as a "recipient" in <xref target="RFC7519"/> or "receiver" in
             related specifications.
           </t>
             
           <t hangText="Subject">
 	    <vspace/>
             A SET describes an event or state change that has occurred 
-            about a Subject. A Subject might, for instance, be a principal (e.g., 
+            about a subject. A subject might, for instance, be a principal (e.g., 

```

Comment:
Subject is a defined term I believe. It should not be lowercased as it is in sec 1.5